src/style/objects.accordion.css

```
@@ -1,22 +1,23 @@
-.accordion {
-  border: 2px solid var(--eds_text_static_icons_dark__secondary);
-  border-radius: 4px;
-}
+/* Parent */
 .accordion > div {
-  border: none;
-  min-height: auto;
-}
-.accordion > div > h4, .accordion > div > p {
-  color: inherit;
-  flex: 1;
-}
-.accordion > div > svg {
-  height: 24px;
-  width: 24px;
-  fill: var(--eds_interactive_primary__resting);
+  border: 1px solid var(--eds_ui_background__medium);
+  border-radius: 2px;
```

Style border according to wireframes.
```suggestion
  border: 2px solid var(--eds_text_static_icons_dark__secondary);
  border-radius: 4px;
```

---

fixed